client/src/components/Status.js

```diff
@@ -1,7 +1,12 @@
 import React, {Component} from 'react';
 import Button from 'react-bootstrap/Button';
+const axios = require('axios');
 
 class Status extends Component {
+    constructor(props) {
+        super(props);
+        axios.post("http://localhost:5000/api/getS3");

```

Comment:
The domain can't be hardcoded or it won't work on heroku. It needed to needs to be `axios.post("/api/getS3")` because the domain is different on heroku.